src/Tabs/TabTemplate.spec.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-env mocha */
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from 'chai';
+import TabTemplate from './TabTemplate';
+import getMuiTheme from '../styles/getMuiTheme';
+
+describe('<TabTemplate />', () => {
+  const muiTheme = getMuiTheme();
+  const shallowWithContext = (node) => shallow(node, {context: {muiTheme}});
+
+  it('should have different tab template style', () => {
+    const wrapper = shallowWithContext(
+      <TabTemplate
+        style={{backgroundColor: 'red'}}
+        selected={false}
+      />
+    );
+
+    expect(wrapper.prop('style')).to.have.property('backgroundColor', 'red');
```

Comment:
Just to make clear, by using `assert.strictEqual`, every time `<TabTemplate />`'s style is changed, we will need to change its unit tests, right?